### PR TITLE
Force installation of aardvark-dns if netavark is installed

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

On SLE Micro 5.5, sometimes we have the netavark package installed but not the aardvark-dns one as it's only a recommends. Without it the server will have issues so force it.

## Test coverage
- No tests: RPM requires

- [X] **DONE**

## Links

Issue(s): https://bugzilla.suse.com/show_bug.cgi?id=1220371

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: uyuni-tools.changes.cbosdonnat.aardvark-dns [hunk 1]
```diff
@@ -0,0 +1 @@
+- Install aardvark-dns is netavark is installed (bsc#1220371)
```
Comment (Bischoff) [inline uyuni-tools.changes.cbosdonnat.aardvark-dns:1]: typo: "if"

**Comment by deneb-alpha** [comment]
@cbosdo I'm  checking something before giving my approval.  I think I need to create  a branch from  the beta1 tag and have this PR rebased against this branch (I need to resubmit uyuni-tools).

but  let me double-check first.

File: uyuni-tools.changes.cbosdonnat.aardvark-dns (hunk 1 above)
Comment (deneb-alpha) [inline uyuni-tools.changes.cbosdonnat.aardvark-dns:1]:
```suggestion
- Install aardvark-dns is netavark if installed (bsc#1220371)
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
could you please  rebase this change against the branch `Manager-5.0-Beta1`?

please, note that the branch `Manager-5.0-Beta1` has been created only now and it was based on the previous tag Manager-5.0-Beta1 from commit edea52682473fefd7376e8445d2b0af439c95a10

I'm not sure if there's anything else needed in the new branch

File: uyuni-tools.changes.cbosdonnat.aardvark-dns (hunk 1 above)
Comment (cbosdo) [reply uyuni-tools.changes.cbosdonnat.aardvark-dns]: Fixed

Comment (cbosdo) [reply uyuni-tools.changes.cbosdonnat.aardvark-dns]: Fixed

**Comment by cbosdo** [comment]
@deneb-alpha rebased on Beta1 branch

**Review by deneb-alpha — APPROVED** [review]
thanks, LGTM

**Pull request merged**